dependencies/phoebus-target/pom.xml

```diff
@@ -10,6 +10,7 @@
   <properties>
     <spring.boot-version>2.0.5.RELEASE</spring.boot-version>
     <preReleaseScripts>${project.basedir}/release_classpath.py</preReleaseScripts>
+    <batik.version>1.12</batik.version>
```

Comment:
Suggest the batik.version should be in the top level (parent) pom, and then also be used in the app/display/representation-javafx/pom.xml

Reply:
Makes sense. Will fix it.